examples/photos.js

```diff
@@ -13,6 +13,33 @@ const createPhotoPiles = async element => {
   piling.set('renderer', imageRenderer);
   piling.set('items', data);
 
+  const handlePileDrop = () => {
+    piling.style(
+      'itemOpacity',
+      (item, pile, i) =>
+        (pile.itemContainer.children.length - i) /
```

Comment:
On a different note, we should start abstracting some of these things away into properties so that the user can simply do `pile.size` where `size` is just a getter that returns `pile.itemContainer.children.length` :)